playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -62,6 +64,10 @@
   roles:
   - openshift_facts
   post_tasks:
+
+  - include: "{{ openshift_upgrade_pre_master_hook }}"
```

Comment:
Sounds like a good idea to me. I have a number of other suggestions for present and future from a long chat with @detiber I need to add in here as well.

Just doing some testing to see how this works for ops right now, and then will return to finalize.